aip/cloud/25001.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 25001
+state: draft
+created: TBD
+placement:
+  category: gcp
+---
+
+# Empty Objects
+
+Fields of type Message **must not** be empty. Empty messages do not have uniform
```

Comment:
I think that this sentence is too broad in its guidance. Empty messages are very useful for future proofing APIs, whether it be a response message, Operation metadata, or just a nested message in a resource. I don't think the use of empty messages should be banned completely. It looks like you have a couple of specific scenarios, I think the guidance should focus on those scenarios, and potentially outline when it is OK to use empty messages (i.e. in the scenarios i've mentioned above).

Reply:
Hm, ok how about something like:

Messages within the body of resource requests or responses must not be empty.

No problems with empty responses as a concept, or empty operation information. Just empty objects within resources are the target